docs/release/trg-0/trg-1-05.md

```diff
@@ -7,30 +7,29 @@ sidebar_position: 1
 Proposed release date: 30th of March 2024
 :::
 
-| Status     | Created      | Post-History                           |
-|------------|--------------|----------------------------------------|
-| Initial version  | 10-Jan-2024  | Transfered from Release Acceptance Criteria into TRG|
-| Draft | 26-March-2024  | Transfered from Release Acceptance Criteria into TRG|
+| Status          | Created     | Post-History                                          |
+|-----------------|-------------|-------------------------------------------------------|
+| Draft           | 26-Mar-2024 | Transferred from Release Acceptance Criteria into TRG |
 
 ## Why
 
 To provide information like context, integration interfaces and planned usage of the component. The architecture documentation should support architects and operation teams for their decisions and operative tasks. We follow the good practice to document the Architecture of FOSS components. [ARC42](https://arc42.org/) is our template of choice.
 
 ## Description
 
-In order to pragmatically start or update your Architecture documentation, consult [ARC42](https://arc42.org/). An "Architecture" folder must be present in the /docs/ folder of your leading repository. Within the Architecture folder the structure of the ARC42 template shall be applied. Follow the guidance to understand, WHAT and HOW to document.
+In order to pragmatically start or update your Architecture documentation, consult [ARC42](https://arc42.org/). An "Architecture" folder must be present in the `/docs/` folder of your leading repository. Within the Architecture folder the structure of the ARC42 template shall be applied. Follow the guidance to understand, WHAT and HOW to document.
```

Comment:
For integrity reasons I'd suggest to store the architectur docs in `/docs/architecture` folder with an explaining readme file in `/docs/README.md`.

Then we have:
- `/docs/README.md`
- `/docs/admin/`
- `/docs/architecture/`
- `/docs/user/`

Reply:
You will need to address this in the draft PR later on. I can't add this without clarifying with the original authors. I do like the suggestion though